pkg/doc/signature/proof/data.go

```diff
@@ -86,14 +110,15 @@ const (
 	proofType excludedKey = iota + 1
 	proofID
 	proofValue
+	jws
 )
 
 func (ek excludedKey) String() string {
-	return [...]string{"type", "id", "proofValue"}[ek-1]
+	return [...]string{"type", "id", "proofValue", "jws"}[ek-1]
```

Comment:
I would make it a global vav  [...]string{"type", "id", "proofValue", "jws"}

Reply:
It's a matter of taste, I think. But in this particular case, I will create two constants for `[...]string{"type", "id", "proofValue"}` and `[...]excludedKey{proofType, proofID, proofValue, jws} ` so that they will be placed together and it will be easier to refine them in future.